fr.openfisca.org/api/requirements.txt

```diff
@@ -0,0 +1,2 @@
+OpenFisca-France >= 18.0.0, < 19.0
```

Comment:
Why not >= 18.10 as in [update-api-cmd branch](https://github.com/openfisca/openfisca-france/tree/update-api-cmd) ?